mitodl/mit-open
# Migrate useMediaQuery hooks

### What are the relevant tickets?

Fixes https://github.com/mitodl/hq/issues/5471

### Description (What does it do?)


Replaces all uses of `useMuiBreakpointAtLeast()` with css-only media queries. The reason is that the hook uses [window.matchMedia()](https://developer.mozilla.org/en-US/docs/Web/API/Window/matchMedia), which is not availalbe in the server for server rendered content.

Also fixes css issues with the headings in the homepage Testimonials and News / Events that I noticed (present in RC and Prod).

Whitelists all image URLs in the config.


### Screenshots (if appropriate):

<img width="321" alt="image" src="https://github.com/user-attachments/assets/1af5fba1-c81c-46b4-b322-f89272d63fe2">


### How can this be tested?


Load the homepage.

- The News and Events section should appear and behave as on prod at all screen widths.

Load the search page

- The search results should display the mobile loading state as smaller screen widths and the desktop loading state at larger screen widths





### Additional Context
